### ISSUE TITLE
RuntimeException 'Cannot execute operation because FlutterJNI is not attached to native' occurs when exit app with add-2-app

### ISSUE THREAD

**Issue by cmengler** [flutter/flutter#53076]
This crash occurs when the Flutter UI is either displaying a platform view (e.g. `WebView`) or video playback -- the example Android host app has been changed to a `FragmentActivity` using a `FlutterFragment` with `destroyEngineWithFragment` set to **true**.

Observations I've made:

**video_player**
- Samsung S10 (Android 10): The application crashes when video playback is in progress.
- Samsung S7 (Android 8): The application does not crash when video playback is in progress, however the audio can still be heard.
- Samsung S10/S7: Pausing the video play back prior to destroying the activity avoids the crash.

**webview_flutter**
- Samsung S10/S7: The application crashes if a WebView is displayed.
- Samsung S10/S7: The application does NOT crash if `flutterEngine.getPlatformViewsController().onFlutterViewDestroyed()` is called `onDestroy` of the activity.

Example app for reproducing: https://github.com/cmengler/android-flutter-crash-on-exit

## Steps to Reproduce

1. Run the app `flutter run`, allow Flutter UI to render.
2. Press back button to exit app.

**Expected results:**

- App close/exits gracefully with no crash.

**Actual results:**

- App crashes with `RuntimeException`

<details>
  <summary>Logs</summary>

```
2020-03-23 15:41:34.494 19741-19741/com.example.android_crash_on_exit E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.example.android_crash_on_exit, PID: 19741
    java.lang.RuntimeException: Cannot execute operation because FlutterJNI is not attached to native.
        at io.flutter.embedding.engine.FlutterJNI.ensureAttachedToNative(FlutterJNI.java:227)
        at io.flutter.embedding.engine.FlutterJNI.markTextureFrameAvailable(FlutterJNI.java:554)
        at io.flutter.embedding.engine.renderer.FlutterRenderer.markTextureFrameAvailable(FlutterRenderer.java:274)
        at io.flutter.embedding.engine.renderer.FlutterRenderer.access$300(FlutterRenderer.java:38)
        at io.flutter.embedding.engine.renderer.FlutterRenderer$SurfaceTextureRegistryEntry$1.onFrameAvailable(FlutterRenderer.java:145)
        at android.graphics.SurfaceTexture$1.handleMessage(SurfaceTexture.java:211)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:237)
        at android.app.ActivityThread.main(ActivityThread.java:7777)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1047)
```

```
[✓] Flutter (Channel unknown, v1.12.13+hotfix.5, on Mac OS X 10.14.5 18F203, locale en-AU)
    • Flutter version 1.12.13+hotfix.5 at /Users/cmengler/Documents/flutter-v1.12.13+hotfix.5
    • Framework revision 27321ebbad (3 months ago), 2019-12-10 18:15:01 -0800
    • Engine revision 2994f7e1e6
    • Dart version 2.7.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at /Users/cmengler/Documents/Android/Sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.0
    • ANDROID_HOME = /Users/cmengler/Documents/Android/Sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3, Build version 11C29
    • CocoaPods version 1.8.4

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 44.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] VS Code (version 1.43.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.8.1

[✓] Connected device (4 available)
    • SM G973F   • RF9U8YJ        • android-arm64  • Android 10 (API 29)
    • SM G930F   • ce121cbc • android-arm64  • Android 8.0.0 (API 26)
    • Chrome     • chrome             • web-javascript • Google Chrome 80.0.3987.149
    • Web Server • web-server         • web-javascript • Flutter Tools

• No issues found!
```
</details>

**Comment by pamir72** [comment]
#48063

**Comment by Omi231** [comment]
@cmengler  Did you find any solution or workaround?

**Comment by cmengler** [comment]
> @cmengler Did you find any solution or workaround?

@Omi231 for `webview_flutter` (or any platform views), the following avoids the crash.

```java
public class MyActivity extends FragmentActivity {
    ...
    @Override
    protected void onDestroy() {
        flutterEngine.getPlatformViewsController().onFlutterViewDestroyed();
        super.onDestroy();
    }
    ...
}
```

For `video_player`, as a workaround for **now** we're pausing the video before the activity is dismissed either by user action or pressing the back button. In our app we are using a platform channel to communicate between the host app and Flutter UI which means we can easily hook into `onBackPressed()` of the activity -- not ideal but will be revisiting this again :)

**Comment by Pankti16** [comment]
> ```java
>  flutterEngine.getPlatformViewsController().onFlutterViewDestroyed();
> ```

Where do I put ``` flutterEngine.getPlatformViewsController().onFlutterViewDestroyed();``` line?

**Comment by FacuRossi** [comment]
same issue. using the camera plugin.
the devices are samsung and xiaomi.
![image](https://user-images.githubusercontent.com/21088004/78940732-b8a43200-7a8c-11ea-8100-a5c7527b3640.png)
any update on this?

**Comment by jidogoon** [comment]
I've faced same issue when using camera plugin.
I already added the code `flutterEngine.getPlatformViewsController().onFlutterViewDestroyed()`, but it wasn't helpful.

In my case, I fixed this problem releasing camera controller before Widget closing.
But I don't think this is a good solution. just FYI.

```
@override
  Widget build(BuildContext context) {
    return WillPopScope(
      onWillPop: () async {
        // in this code, CameraController disposes.
        await CameraManager.instance.releaseCamera();
        return true;
      },
      child: ...
```

**Comment by kw2019ltd** [comment]
hi
getting below issue with Flutter (Channel unknown, v1.17.1)


Fatal Exception: java.lang.RuntimeException: Cannot execute operation because FlutterJNI is not attached to native.
       at io.flutter.embedding.engine.FlutterJNI.ensureAttachedToNative(FlutterJNI.java:222)
       at io.flutter.embedding.engine.FlutterJNI.markTextureFrameAvailable(FlutterJNI.java:543)
       at io.flutter.embedding.engine.renderer.FlutterRenderer.markTextureFrameAvailable(FlutterRenderer.java:295)
       at io.flutter.embedding.engine.renderer.FlutterRenderer.access$300(FlutterRenderer.java:35)
       at io.flutter.embedding.engine.renderer.FlutterRenderer$SurfaceTextureRegistryEntry$1.onFrameAvailable(FlutterRenderer.java:141)
       at android.graphics.SurfaceTexture$1.handleMessage(SurfaceTexture.java:203)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5421)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)

image

$ flutter doctor -v
[✓] Flutter (Channel unknown, v1.17.1, on Linux, locale en_US.UTF-8)
• Flutter version 1.17.1 at /home/cirrus/sdks/flutter
• Framework revision f7a6a79 (3 days ago), 2020-05-12 18:39:00 -0700
• Engine revision 6bc433c6b6
• Dart version 2.8.2
[!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
• Android SDK at /opt/android-sdk-linux
• Platform android-29, build-tools 29.0.2
• ANDROID_HOME = /opt/android-sdk-linux
• ANDROID_SDK_ROOT = /opt/android-sdk-linux
• Java binary at: /usr/bin/java
• Java version OpenJDK Runtime Environment (build 1.8.0_222-8u222-b10-1ubuntu1~18.04.1-b10)
! Some Android licenses not accepted. To resolve this, run: flutter doctor --android-licenses
[!] Android Studio (not installed)
• Android Studio not found; download from https://developer.android.com/studio/index.html
(or visit https://flutter.dev/docs/get-started/install/linux#android-setup for detailed instructions).
[!] Connected device
! No devices available
! Doctor found issues in 3 categories

**Comment by Omi231** [comment]
> > @cmengler Did you find any solution or workaround?
> 
> @Omi231 for `webview_flutter` (or any platform views), the following avoids the crash.
> 
> ```java
> public class MyActivity extends FragmentActivity {
>     ...
>     @Override
>     protected void onDestroy() {
>         flutterEngine.getPlatformViewsController().onFlutterViewDestroyed();
>         super.onDestroy();
>     }
>     ...
> }
> ```
> 
> For `video_player`, as a workaround for **now** we're pausing the video before the activity is dismissed either by user action or pressing the back button. In our app we are using a platform channel to communicate between the host app and Flutter UI which means we can easily hook into `onBackPressed()` of the activity -- not ideal but will be revisiting this again :)

It did fix the crash problem but now after upgrading to flutter 1.18.0, the added code is giving error and when this code is removed from the activity file it runs ok. Does it mean this issue is fixed in 1.18 and removing this temporary fix will not crash the app in production version?

**Comment by chillbrodev** [comment]
> I've faced same issue when using camera plugin.
> I already added the code `flutterEngine.getPlatformViewsController().onFlutterViewDestroyed()`, but it wasn't helpful.
> 
> In my case, I fixed this problem releasing camera controller before Widget closing.
> But I don't think this is a good solution. just FYI.
> 
> ```
> @override
>   Widget build(BuildContext context) {
>     return WillPopScope(
>       onWillPop: () async {
>         // in this code, CameraController disposes.
>         await CameraManager.instance.releaseCamera();
>         return true;
>       },
>       child: ...
> ```

@jidogoon Thank you so much! this totally fixed my issue. Let me buy you a coffee.

**Comment by pedromassangocode** [comment]
Hi @cmengler 
I cannot reproduce this with the latest `stable` version of Flutter, tested on Xiaomi Redmi 5Plus.

After pressing back it still shows logs but they are unrelated to this issue!
Please try the latest stable channel and verify if the issue sill persists.

<details><summary>Logs</summary>

```
Launching lib/main.dart on Redmi 5 Plus in debug mode...
Running Gradle task 'assembleDebug'...
✓ Built build/app/outputs/flutter-apk/app-debug.apk.
Waiting for Redmi 5 Plus to report its views...
Debug service listening on ws://127.0.0.1:62926/hrm9WAeZpKM=/ws
Syncing files to device Redmi 5 Plus...
W/PlatformViewsController(17804): Creating a virtual display of size: [1080, 2160] may result in problems(https://github.com/flutter/flutter/issues/2897).It is larger than the device screen size: [1080, 2030].
W/ResourceType(17804): No package identifier when getting name for resource number 0x00000000
W/ResourceType(17804): No package identifier when getting name for resource number 0x00000000
W/ResourceType(17804): Found multiple library tables, ignoring...
I/WebViewFactory(17804): Loading com.android.chrome version 86.0.4240.198 (code 424019823)
W/zygote64(17804): miui_dex2oat:　OatFile: /data/app/com.android.chrome-MzqpDWs6S-pZCDc3YtqDJQ==/oat/arm64/base.odex Compiler-Filter = speed
W/zygote64(17804): miui_dex2oat:　OatFile: /data/app/com.android.chrome-MzqpDWs6S-pZCDc3YtqDJQ==/oat/arm64/split_autofill_assistant.odex Compiler-Filter = speed
W/zygote64(17804): miui_dex2oat: oat file of /data/app/com.android.chrome-MzqpDWs6S-pZCDc3YtqDJQ==/split_autofill_assistant.config.en.apk is not exists
W/zygote64(17804): miui_dex2oat: oat file of /data/app/com.android.chrome-MzqpDWs6S-pZCDc3YtqDJQ==/split_config.en.apk is not exists
W/zygote64(17804): miui_dex2oat:　OatFile: /data/app/com.android.chrome-MzqpDWs6S-pZCDc3YtqDJQ==/oat/arm64/split_extra_icu.odex Compiler-Filter = speed
W/ResourceType(17804): No package identifier when getting name for resource number 0x00000000
I/chatty  (17804): uid=10176(com.example.android_crash_on_exit) identical 1 line
W/ResourceType(17804): No package identifier when getting name for resource number 0x00000000
I/cr_LibraryLoader(17804): Loaded native library version number "86.0.4240.198"
I/cr_CachingUmaRecorder(17804): Flushed 5 samples from 5 histograms.
I/zygote64(17804): Do partial code cache collection, code=30KB, data=21KB
I/zygote64(17804): After code cache collection, code=30KB, data=21KB
I/zygote64(17804): Increasing code cache capacity to 128KB
W/ResourceType(17804): Failure getting entry for 0x7f13060f (t=18 e=1551) (error -2147483647)
D/EgretLoader(17804): EgretLoader(Context context)
D/EgretLoader(17804): The context is not activity
W/cr_media(17804): Requires BLUETOOTH permission
W/ContentCatcher(17804): Failed to notify a WebView
I/zygote64(17804): Do partial code cache collection, code=61KB, data=48KB
I/zygote64(17804): After code cache collection, code=61KB, data=48KB
I/zygote64(17804): Increasing code cache capacity to 256KB
W/Looper  (17804): Slow Frame: doFrame is 344ms late
W/VideoCapabilities(17804): Unrecognized profile 2130706433 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706434 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706433 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706434 for video/avc
W/VideoCapabilities(17804): Unsupported mime video/divx
W/VideoCapabilities(17804): Unsupported mime video/divx311
W/VideoCapabilities(17804): Unsupported mime video/divx4
W/VideoCapabilities(17804): Unrecognized profile/level 0/3 for video/mpeg2
W/VideoCapabilities(17804): Unrecognized profile/level 0/3 for video/mpeg2
W/VideoCapabilities(17804): Unsupported mime video/x-ms-wmv
W/VideoCapabilities(17804): Unsupported mime video/x-ms-wmv
W/VideoCapabilities(17804): Unsupported mime video/x-ms-wmv
W/VideoCapabilities(17804): Unrecognized profile 2130706433 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706434 for video/avc
I/VideoCapabilities(17804): Unsupported profile 4 for video/mp4v-es
W/Utils   (17804): could not parse long range '175-174'
W/VideoCapabilities(17804): Unrecognized profile 2130706433 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706434 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706433 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706434 for video/avc
W/VideoCapabilities(17804): Unsupported mime video/divx
W/VideoCapabilities(17804): Unsupported mime video/divx311
W/VideoCapabilities(17804): Unsupported mime video/divx4
W/VideoCapabilities(17804): Unrecognized profile/level 0/3 for video/mpeg2
W/VideoCapabilities(17804): Unrecognized profile/level 0/3 for video/mpeg2
W/VideoCapabilities(17804): Unsupported mime video/x-ms-wmv
I/chatty  (17804): uid=10176(com.example.android_crash_on_exit) ThreadPoolSingl identical 1 line
W/VideoCapabilities(17804): Unsupported mime video/x-ms-wmv
W/VideoCapabilities(17804): Unrecognized profile 2130706433 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706434 for video/avc
I/VideoCapabilities(17804): Unsupported profile 4 for video/mp4v-es
W/Utils   (17804): could not parse long range '175-174'
I/cr_MediaCodecBridge(17804): create MediaCodec video decoder, mime video/avc
I/cr_MediaCodecBridge(17804): create MediaCodec video decoder, mime video/avc
I/OMXClient(17804): Treble IOmx obtained
I/OMXClient(17804): Treble IOmx obtained
W/VideoCapabilities(17804): Unrecognized profile 2130706433 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706434 for video/avc
I/MediaCodec(17804): MediaCodec will operate in async mode
I/MediaCodec(17804): MediaCodec will operate in async mode
D/SurfaceUtils(17804): connecting to surface 0x712756b010, reason connectToSurface
D/SurfaceUtils(17804): connecting to surface 0x7128a41010, reason connectToSurface
I/MediaCodec(17804): [OMX.qcom.video.decoder.avc] setting surface generation to 18231297
I/MediaCodec(17804): [OMX.google.h264.decoder] setting surface generation to 18231298
D/SurfaceUtils(17804): disconnecting from surface 0x7128a41010, reason connectToSurface(reconnect)
D/SurfaceUtils(17804): disconnecting from surface 0x712756b010, reason connectToSurface(reconnect)
D/SurfaceUtils(17804): connecting to surface 0x712756b010, reason connectToSurface(reconnect)
D/SurfaceUtils(17804): connecting to surface 0x7128a41010, reason connectToSurface(reconnect)
E/ACodec  (17804): [OMX.google.h264.decoder] setPortMode on output to DynamicANWBuffer failed w/ err -1010
I/ExtendedACodec(17804): setupVideoDecoder()
I/ExtendedACodec(17804): setupVideoDecoder()
W/ExtendedACodec(17804): Failed to get extension for extradata parameter
I/ExtendedACodec(17804): Decoder will be in frame by frame mode
D/SurfaceUtils(17804): set up nativeWindow 0x7128a41010 for 1080x820, color 0x7fa30c06, rotation 0, usage 0x20002900
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
I/cr_MediaCodecBridge(17804): create MediaCodec video decoder, mime video/avc
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
I/chatty  (17804): uid=10176(com.example.android_crash_on_exit) CodecLooper identical 1 line
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
I/OMXClient(17804): Treble IOmx obtained
D/SurfaceUtils(17804): set up nativeWindow 0x7128a41010 for 1088x832, color 0x7fa30c06, rotation 0, usage 0x20002900
W/VideoCapabilities(17804): Unrecognized profile 2130706433 for video/avc
W/VideoCapabilities(17804): Unrecognized profile 2130706434 for video/avc
I/MediaCodec(17804): MediaCodec will operate in async mode
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
D/SurfaceUtils(17804): connecting to surface 0x7128a42010, reason connectToSurface
I/MediaCodec(17804): [OMX.qcom.video.decoder.avc] setting surface generation to 18231299
D/SurfaceUtils(17804): disconnecting from surface 0x7128a42010, reason connectToSurface(reconnect)
D/SurfaceUtils(17804): connecting to surface 0x7128a42010, reason connectToSurface(reconnect)
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
I/ExtendedACodec(17804): setupVideoDecoder()
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
I/chatty  (17804): uid=10176(com.example.android_crash_on_exit) CodecLooper identical 1 line
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
I/ExtendedACodec(17804): Decoder will be in frame by frame mode
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
D/SurfaceUtils(17804): set up nativeWindow 0x7128a42010 for 1080x820, color 0x7fa30c06, rotation 0, usage 0x20002900
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
I/chatty  (17804): uid=10176(com.example.android_crash_on_exit) CodecLooper identical 1 line
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
D/SurfaceUtils(17804): set up nativeWindow 0x7128a42010 for 1088x832, color 0x7fa30c06, rotation 0, usage 0x20002900
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
I/chatty  (17804): uid=10176(com.example.android_crash_on_exit) CodecLooper identical 6 lines
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
D/MediaCodec(17804): [OMX.google.h264.decoder] setting dataspace on output surface to #104
W/GrallocMapperPassthrough(17804): buffer descriptor with invalid usage bits 0x2000
D/SoftwareRenderer(17804): setting dataspace on output surface to #104
I/zygote64(17804): Rejecting re-init on previously-failed class java.lang.Class<androidx.core.view.ViewCompat$2>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/View$OnUnhandledKeyEventListener;
I/zygote64(17804):   at boolean androidx.core.view.ViewCompat.dispatchUnhandledKeyEventBeforeHierarchy(android.view.View, android.view.KeyEvent) (ViewCompat.java:3750)
I/zygote64(17804):   at boolean androidx.core.view.KeyEventDispatcher.dispatchBeforeHierarchy(android.view.View, android.view.KeyEvent) (KeyEventDispatcher.java:63)
I/zygote64(17804):   at boolean androidx.core.app.ComponentActivity.dispatchKeyEvent(android.view.KeyEvent) (ComponentActivity.java:130)
I/zygote64(17804):   at boolean com.android.internal.policy.DecorView.dispatchKeyEvent(android.view.KeyEvent) (DecorView.java:357)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4819)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4691)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4377)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4434)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4410)
I/zygote64(17804):   at void android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(java.lang.Object, boolean) (ViewRootImpl.java:4573)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$PendingEvent.run() (InputMethodManager.java:2430)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(android.view.inputmethod.InputMethodManager$PendingEvent, boolean) (InputMethodManager.java:1993)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.finishedInputEvent(int, boolean, boolean) (InputMethodManager.java:1984)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(int, boolean) (InputMethodManager.java:2407)
I/zygote64(17804):   at void android.view.InputEventSender.dispatchInputEventFinished(int, boolean) (InputEventSender.java:141)
I/zygote64(17804):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
I/zygote64(17804):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:331)
I/zygote64(17804):   at void android.os.Looper.loop() (Looper.java:149)
I/zygote64(17804):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6651)
I/zygote64(17804):   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/zygote64(17804):   at void com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run() (RuntimeInit.java:547)
I/zygote64(17804):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:824)
I/zygote64(17804): Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.View$OnUnhandledKeyEventListener" on path: DexPathList[[zip file "/data/app/com.example.android_crash_on_exit-XlDV9N0buu3z756eMOEL5g==/base.apk"],nativeLibraryDirectories=[/data/app/com.example.android_crash_on_exit-XlDV9N0buu3z756eMOEL5g==/lib/arm64, /data/app/com.example.android_crash_on_exit-XlDV9N0buu3z756eMOEL5g==/base.apk!/lib/arm64-v8a, /system/lib64, /system/vendor/lib64]]
I/zygote64(17804):   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:125)
I/zygote64(17804):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:379)
I/zygote64(17804):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
I/zygote64(17804):   at boolean androidx.core.view.ViewCompat.dispatchUnhandledKeyEventBeforeHierarchy(android.view.View, android.view.KeyEvent) (ViewCompat.java:3750)
I/zygote64(17804):   at boolean androidx.core.view.KeyEventDispatcher.dispatchBeforeHierarchy(android.view.View, android.view.KeyEvent) (KeyEventDispatcher.java:63)
I/zygote64(17804):   at boolean androidx.core.app.ComponentActivity.dispatchKeyEvent(android.view.KeyEvent) (ComponentActivity.java:130)
I/zygote64(17804):   at boolean com.android.internal.policy.DecorView.dispatchKeyEvent(android.view.KeyEvent) (DecorView.java:357)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4819)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4691)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4377)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4434)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4410)
I/zygote64(17804):   at void android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(java.lang.Object, boolean) (ViewRootImpl.java:4573)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$PendingEvent.run() (InputMethodManager.java:2430)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(android.view.inputmethod.InputMethodManager$PendingEvent, boolean) (InputMethodManager.java:1993)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.finishedInputEvent(int, boolean, boolean) (InputMethodManager.java:1984)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(int, boolean) (InputMethodManager.java:2407)
I/zygote64(17804):   at void android.view.InputEventSender.dispatchInputEventFinished(int, boolean) (InputEventSender.java:141)
I/zygote64(17804):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
I/zygote64(17804):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:331)
I/zygote64(17804):   at void android.os.Looper.loop() (Looper.java:149)
I/zygote64(17804):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6651)
I/zygote64(17804):   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/zygote64(17804):   at void com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run() (RuntimeInit.java:547)
I/zygote64(17804):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:824)
I/zygote64(17804): 
I/zygote64(17804): Rejecting re-init on previously-failed class java.lang.Class<androidx.core.view.ViewCompat$2>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/View$OnUnhandledKeyEventListener;
I/zygote64(17804):   at boolean androidx.core.view.ViewCompat.dispatchUnhandledKeyEventBeforeHierarchy(android.view.View, android.view.KeyEvent) (ViewCompat.java:3750)
I/zygote64(17804):   at boolean androidx.core.view.KeyEventDispatcher.dispatchBeforeHierarchy(android.view.View, android.view.KeyEvent) (KeyEventDispatcher.java:63)
I/zygote64(17804):   at boolean androidx.core.app.ComponentActivity.dispatchKeyEvent(android.view.KeyEvent) (ComponentActivity.java:130)
I/zygote64(17804):   at boolean com.android.internal.policy.DecorView.dispatchKeyEvent(android.view.KeyEvent) (DecorView.java:357)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4819)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4691)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4377)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4434)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4410)
I/zygote64(17804):   at void android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(java.lang.Object, boolean) (ViewRootImpl.java:4573)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$PendingEvent.run() (InputMethodManager.java:2430)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(android.view.inputmethod.InputMethodManager$PendingEvent, boolean) (InputMethodManager.java:1993)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.finishedInputEvent(int, boolean, boolean) (InputMethodManager.java:1984)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(int, boolean) (InputMethodManager.java:2407)
I/zygote64(17804):   at void android.view.InputEventSender.dispatchInputEventFinished(int, boolean) (InputEventSender.java:141)
I/zygote64(17804):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
I/zygote64(17804):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:331)
I/zygote64(17804):   at void android.os.Looper.loop() (Looper.java:149)
I/zygote64(17804):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6651)
I/zygote64(17804):   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/zygote64(17804):   at void com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run() (RuntimeInit.java:547)
I/zygote64(17804):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:824)
I/zygote64(17804): Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.View$OnUnhandledKeyEventListener" on path: DexPathList[[zip file "/data/app/com.example.android_crash_on_exit-XlDV9N0buu3z756eMOEL5g==/base.apk"],nativeLibraryDirectories=[/data/app/com.example.android_crash_on_exit-XlDV9N0buu3z756eMOEL5g==/lib/arm64, /data/app/com.example.android_crash_on_exit-XlDV9N0buu3z756eMOEL5g==/base.apk!/lib/arm64-v8a, /system/lib64, /system/vendor/lib64]]
I/zygote64(17804):   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:125)
I/zygote64(17804):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:379)
I/zygote64(17804):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
I/zygote64(17804):   at boolean androidx.core.view.ViewCompat.dispatchUnhandledKeyEventBeforeHierarchy(android.view.View, android.view.KeyEvent) (ViewCompat.java:3750)
I/zygote64(17804):   at boolean androidx.core.view.KeyEventDispatcher.dispatchBeforeHierarchy(android.view.View, android.view.KeyEvent) (KeyEventDispatcher.java:63)
I/zygote64(17804):   at boolean androidx.core.app.ComponentActivity.dispatchKeyEvent(android.view.KeyEvent) (ComponentActivity.java:130)
I/zygote64(17804):   at boolean com.android.internal.policy.DecorView.dispatchKeyEvent(android.view.KeyEvent) (DecorView.java:357)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4819)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4691)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4377)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4434)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4410)
I/zygote64(17804):   at void android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(java.lang.Object, boolean) (ViewRootImpl.java:4573)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$PendingEvent.run() (InputMethodManager.java:2430)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(android.view.inputmethod.InputMethodManager$PendingEvent, boolean) (InputMethodManager.java:1993)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.finishedInputEvent(int, boolean, boolean) (InputMethodManager.java:1984)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(int, boolean) (InputMethodManager.java:2407)
I/zygote64(17804):   at void android.view.InputEventSender.dispatchInputEventFinished(int, boolean) (InputEventSender.java:141)
I/zygote64(17804):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
I/zygote64(17804):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:331)
I/zygote64(17804):   at void android.os.Looper.loop() (Looper.java:149)
I/zygote64(17804):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6651)
I/zygote64(17804):   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/zygote64(17804):   at void com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run() (RuntimeInit.java:547)
I/zygote64(17804):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:824)
I/zygote64(17804): 
I/zygote64(17804): Rejecting re-init on previously-failed class java.lang.Class<androidx.core.view.ViewCompat$2>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/View$OnUnhandledKeyEventListener;
I/zygote64(17804):   at boolean androidx.core.view.ViewCompat.dispatchUnhandledKeyEventBeforeHierarchy(android.view.View, android.view.KeyEvent) (ViewCompat.java:3750)
I/zygote64(17804):   at boolean androidx.core.view.KeyEventDispatcher.dispatchBeforeHierarchy(android.view.View, android.view.KeyEvent) (KeyEventDispatcher.java:63)
I/zygote64(17804):   at boolean androidx.core.app.ComponentActivity.dispatchKeyEvent(android.view.KeyEvent) (ComponentActivity.java:130)
I/zygote64(17804):   at boolean com.android.internal.policy.DecorView.dispatchKeyEvent(android.view.KeyEvent) (DecorView.java:357)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4819)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4691)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4377)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4434)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4410)
I/zygote64(17804):   at void android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(java.lang.Object, boolean) (ViewRootImpl.java:4573)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$PendingEvent.run() (InputMethodManager.java:2430)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(android.view.inputmethod.InputMethodManager$PendingEvent, boolean) (InputMethodManager.java:1993)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.finishedInputEvent(int, boolean, boolean) (InputMethodManager.java:1984)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(int, boolean) (InputMethodManager.java:2407)
I/zygote64(17804):   at void android.view.InputEventSender.dispatchInputEventFinished(int, boolean) (InputEventSender.java:141)
I/zygote64(17804):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
I/zygote64(17804):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:331)
I/zygote64(17804):   at void android.os.Looper.loop() (Looper.java:149)
I/zygote64(17804):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6651)
I/zygote64(17804):   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/zygote64(17804):   at void com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run() (RuntimeInit.java:547)
I/zygote64(17804):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:824)
I/zygote64(17804): Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.View$OnUnhandledKeyEventListener" on path: DexPathList[[zip file "/data/app/com.example.android_crash_on_exit-XlDV9N0buu3z756eMOEL5g==/base.apk"],nativeLibraryDirectories=[/data/app/com.example.android_crash_on_exit-XlDV9N0buu3z756eMOEL5g==/lib/arm64, /data/app/com.example.android_crash_on_exit-XlDV9N0buu3z756eMOEL5g==/base.apk!/lib/arm64-v8a, /system/lib64, /system/vendor/lib64]]
I/zygote64(17804):   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:125)
I/zygote64(17804):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:379)
I/zygote64(17804):   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
I/zygote64(17804):   at boolean androidx.core.view.ViewCompat.dispatchUnhandledKeyEventBeforeHierarchy(android.view.View, android.view.KeyEvent) (ViewCompat.java:3750)
I/zygote64(17804):   at boolean androidx.core.view.KeyEventDispatcher.dispatchBeforeHierarchy(android.view.View, android.view.KeyEvent) (KeyEventDispatcher.java:63)
I/zygote64(17804):   at boolean androidx.core.app.ComponentActivity.dispatchKeyEvent(android.view.KeyEvent) (ComponentActivity.java:130)
I/zygote64(17804):   at boolean com.android.internal.policy.DecorView.dispatchKeyEvent(android.view.KeyEvent) (DecorView.java:357)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4819)
I/zygote64(17804):   at int android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4691)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4377)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4434)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.apply(android.view.ViewRootImpl$QueuedInputEvent, int) (ViewRootImpl.java:4258)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.deliver(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4231)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.onDeliverToNext(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4284)
I/zygote64(17804):   at void android.view.ViewRootImpl$InputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4250)
I/zygote64(17804):   at void android.view.ViewRootImpl$AsyncInputStage.forward(android.view.ViewRootImpl$QueuedInputEvent) (ViewRootImpl.java:4410)
I/zygote64(17804):   at void android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(java.lang.Object, boolean) (ViewRootImpl.java:4573)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$PendingEvent.run() (InputMethodManager.java:2430)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(android.view.inputmethod.InputMethodManager$PendingEvent, boolean) (InputMethodManager.java:1993)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager.finishedInputEvent(int, boolean, boolean) (InputMethodManager.java:1984)
I/zygote64(17804):   at void android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(int, boolean) (InputMethodManager.java:2407)
I/zygote64(17804):   at void android.view.InputEventSender.dispatchInputEventFinished(int, boolean) (InputEventSender.java:141)
I/zygote64(17804):   at void android.os.MessageQueue.nativePollOnce(long, int) (MessageQueue.java:-2)
I/zygote64(17804):   at android.os.Message android.os.MessageQueue.next() (MessageQueue.java:331)
I/zygote64(17804):   at void android.os.Looper.loop() (Looper.java:149)
I/zygote64(17804):   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6651)
I/zygote64(17804):   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
I/zygote64(17804):   at void com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run() (RuntimeInit.java:547)
I/zygote64(17804):   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:824)
I/zygote64(17804): 
W/InputEventReceiver(17804): Slow Input: took 105ms in dispatching, now at finishInputEvent (KeyEvent: event_seq=0, seq=227486, code=KEYCODE_BACK, action=ACTION_DOWN)
I/chromium(17804): [INFO:CONSOLE(257)] "The deviceorientation events are blocked by feature policy. See https://github.com/WICG/feature-policy/blob/master/features.md#sensor-features", source: https://www.youtube.com/s/player/408be03a/player_ias.vflset/en_US/base.js (257)
W/InputTransport(17804): Slow Input: 231ms so far, channel 'ClientState{5c86e8d uid 10176 pid 17804} (server)' publisher ~ publishKeyEvent: seq=2, deviceId=-1, source=0x101, action=0x1, flags=0x8, keyCode=4, scanCode=0, metaState=0x0, repeatCount=0,downTime=64791764000000, eventTime=64791764000000
I/chromium(17804): [INFO:CONSOLE(257)] "The deviceorientation events are blocked by feature policy. See https://github.com/WICG/feature-policy/blob/master/features.md#sensor-features", source: https://www.youtube.com/s/player/408be03a/player_ias.vflset/en_US/base.js (257)
E/chromium(17804): [ERROR:web_contents_delegate.cc(224)] WebContentsDelegate::CheckMediaAccessPermission: Not supported.
E/chromium(17804): [ERROR:web_contents_delegate.cc(224)] WebContentsDelegate::CheckMediaAccessPermission: Not supported.
W/cr_media(17804): Requires MODIFY_AUDIO_SETTINGS and RECORD_AUDIO. No audio device will be available for recording
I/CameraManagerGlobal(17804): Connecting to camera service
E/libc    (17804): Access denied finding property "camera.hal1.packagelist"
W/ThreadPoolSingl(17804): type=1400 audit(0.0:8429): avc: denied { read } for name="u:object_r:camera_prop:s0" dev="tmpfs" ino=5211 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:camera_prop:s0 tclass=file permissive=0
W/CameraBase(17804): An error occurred while connecting to camera 1: Status(-8): '1: validateClientPermissionsLocked:943: Caller "com.example.android_crash_on_exit" (PID 10176, UID 17804) cannot open camera "1" without camera permission'
E/cr_VideoCapture(17804): Camera.open: 
E/cr_VideoCapture(17804): java.lang.RuntimeException: Fail to connect to camera service
E/cr_VideoCapture(17804): 	at android.hardware.Camera.<init>(Camera.java:608)
E/cr_VideoCapture(17804): 	at android.hardware.Camera.open(Camera.java:434)
E/cr_VideoCapture(17804): 	at org.chromium.media.VideoCaptureFactory.isPanTiltZoomSupported(chromium-Monochrome.aab-stable-424019823:3)
E/libc    (17804): Access denied finding property "camera.hal1.packagelist"
W/ThreadPoolSingl(17804): type=1400 audit(0.0:8430): avc: denied { read } for name="u:object_r:camera_prop:s0" dev="tmpfs" ino=5211 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:camera_prop:s0 tclass=file permissive=0
W/CameraBase(17804): An error occurred while connecting to camera 1: Status(-8): '1: validateClientPermissionsLocked:943: Caller "com.example.android_crash_on_exit" (PID 10176, UID 17804) cannot open camera "1" without camera permission'
E/cr_VideoCapture(17804): Camera.open: 
E/cr_VideoCapture(17804): java.lang.RuntimeException: Fail to connect to camera service
E/cr_VideoCapture(17804): 	at android.hardware.Camera.<init>(Camera.java:608)
E/cr_VideoCapture(17804): 	at android.hardware.Camera.open(Camera.java:434)
E/cr_VideoCapture(17804): 	at org.chromium.media.VideoCaptureFactory.getDeviceSupportedFormats(chromium-Monochrome.aab-stable-424019823:3)
W/ContentCatcher(17804): Failed to notify a WebView
I/zygote64(17804): Do full code cache collection, code=123KB, data=88KB
I/zygote64(17804): After code cache collection, code=101KB, data=59KB
W/ThreadPoolSingl(17804): type=1400 audit(0.0:8431): avc: denied { read } for name="u:object_r:camera_prop:s0" dev="tmpfs" ino=5211 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:camera_prop:s0 tclass=file permissive=0
E/libc    (17804): Access denied finding property "camera.hal1.packagelist"
W/CameraBase(17804): An error occurred while connecting to camera 0: Status(-8): '1: validateClientPermissionsLocked:943: Caller "com.example.android_crash_on_exit" (PID 10176, UID 17804) cannot open camera "0" without camera permission'
E/cr_VideoCapture(17804): Camera.open: 
E/cr_VideoCapture(17804): java.lang.RuntimeException: Fail to connect to camera service
E/cr_VideoCapture(17804): 	at android.hardware.Camera.<init>(Camera.java:608)
E/cr_VideoCapture(17804): 	at android.hardware.Camera.open(Camera.java:434)
E/cr_VideoCapture(17804): 	at org.chromium.media.VideoCaptureFactory.isPanTiltZoomSupported(chromium-Monochrome.aab-stable-424019823:3)
W/ThreadPoolSingl(17804): type=1400 audit(0.0:8432): avc: denied { read } for name="u:object_r:camera_prop:s0" dev="tmpfs" ino=5211 scontext=u:r:untrusted_app:s0:c512,c768 tcontext=u:object_r:camera_prop:s0 tclass=file permissive=0
E/libc    (17804): Access denied finding property "camera.hal1.packagelist"
W/CameraBase(17804): An error occurred while connecting to camera 0: Status(-8): '1: validateClientPermissionsLocked:943: Caller "com.example.android_crash_on_exit" (PID 10176, UID 17804) cannot open camera "0" without camera permission'
E/cr_VideoCapture(17804): Camera.open: 
E/cr_VideoCapture(17804): java.lang.RuntimeException: Fail to connect to camera service
E/cr_VideoCapture(17804): 	at android.hardware.Camera.<init>(Camera.java:608)
E/cr_VideoCapture(17804): 	at android.hardware.Camera.open(Camera.java:434)
E/cr_VideoCapture(17804): 	at org.chromium.media.VideoCaptureFactory.getDeviceSupportedFormats(chromium-Monochrome.aab-stable-424019823:3)
W/cr_MediaCodecBridge(17804): Releasing: OMX.qcom.video.decoder.avc
D/SurfaceUtils(17804): disconnecting from surface 0x7128a41010, reason disconnectFromSurface
W/MediaAnalyticsItem(17804): Unable to record: (codec:0:-1:-11:0:11:android.media.mediacodec.mime=video/avc:android.media.mediacodec.mode=video:android.media.mediacodec.encoder=0:android.media.mediacodec.codec=OMX.qcom.video.decoder.avc:android.media.mediacodec.secure=0:android.media.mediacodec.width=1080:android.media.mediacodec.height=820:android.media.mediacodec.rotation-degrees=0:android.media.mediacodec.maxwidth=1080:android.media.mediacodec.maxheight=820:android.media.mediacodec.bytesin=24564:) [forcenew=0]
W/cr_MediaCodecBridge(17804): Codec released
W/cr_MediaCodecBridge(17804): Releasing: OMX.google.h264.decoder
D/SurfaceUtils(17804): disconnecting from surface 0x712756b010, reason disconnectFromSurface
W/MediaAnalyticsItem(17804): Unable to record: (codec:0:-1:-11:0:10:android.media.mediacodec.codec=OMX.google.h264.decoder:android.media.mediacodec.mode=video:android.media.mediacodec.secure=0:android.media.mediacodec.width=1080:android.media.mediacodec.height=820:android.media.mediacodec.rotation-degrees=0:android.media.mediacodec.maxwidth=1080:android.media.mediacodec.maxheight=820:android.media.mediacodec.mime=video/avc:android.media.mediacodec.bytesin=282414:) [forcenew=0]
W/cr_MediaCodecBridge(17804): Codec released
W/cr_MediaCodecBridge(17804): Releasing: OMX.qcom.video.decoder.avc
D/SurfaceUtils(17804): disconnecting from surface 0x7128a42010, reason disconnectFromSurface
W/MediaAnalyticsItem(17804): Unable to record: (codec:0:-1:-11:0:11:android.media.mediacodec.mime=video/avc:android.media.mediacodec.mode=video:android.media.mediacodec.encoder=0:android.media.mediacodec.codec=OMX.qcom.video.decoder.avc:android.media.mediacodec.secure=0:android.media.mediacodec.width=1080:android.media.mediacodec.height=820:android.media.mediacodec.rotation-degrees=0:android.media.mediacodec.maxwidth=1080:android.media.mediacodec.maxheight=820:android.media.mediacodec.bytesin=135603:) [forcenew=0]
W/cr_MediaCodecBridge(17804): Codec released
W/chromium(17804): [WARNING:spdy_session.cc(3384)] Received HEADERS for invalid stream 7
Lost connection to device.
```
</details>

<details><summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel stable, 1.22.4, on Mac OS X 10.15.7 19H2 darwin-x64, locale en)
    • Flutter version 1.22.4 at /Users/pedromassango/dev/SDKs/flutter_stable
    • Framework revision 1aafb3a8b9 (3 weeks ago), 2020-11-13 09:59:28 -0800
    • Engine revision 2c956a31c0
    • Dart version 2.10.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/pedromassango/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.1, Build version 12A7403
    • CocoaPods version 1.9.3

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Community Edition (version 2020.2.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin installed
    • Dart plugin version 202.8070

[✓] VS Code (version 1.51.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.16.0

[!] Connected device
    ! No devices available

! Doctor found issues in 2 categories.

```
</details>

**Comment by cmengler** [comment]
Cheers @pedromassangocode -- I'll re-check these on my devices with latest stable and will report back here / close the ticket if resolved.

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
